src/generate.js

```diff
@@ -62,7 +62,18 @@ export const generateCSSRuleset = (selector, declarations, stringHandlers,
 
     const prefixedDeclarations = prefixAll(handledDeclarations);
 
-    const rules = objectToPairs(prefixedDeclarations).map(([key, value]) => {
+    const prefixedRules = flatten(
+        objectToPairs(prefixedDeclarations).map(([key, value]) => {
+            if (typeof value === 'object') {
```

Comment:
Yeah I thought of this but I don't know what your polyfilling setup is. If it's safe, it's way better.